include/quicly/cc.h

```diff
@@ -0,0 +1,38 @@
+/* Interface definition for quicly's congestion controller.
+ */
+
+#ifndef quicly_cc_h
+#define quicly_cc_h
+
+#include "quicly/constants.h"
+#include <assert.h>
+#include <stdint.h>
+#include <string.h>
+
+struct ccstate {
+    uint32_t cwnd;
+    uint32_t ssthresh;
+    uint32_t stash;
+    uint8_t recovery_end;
```

Comment:
Is this a PN? Then I think it needs to be uint64_t.

Reply:
I had it as a boolean, and then changed it to a PN. Thanks for catching this.